lib/tool_belt/commands/procedure.rb

```diff
@@ -0,0 +1,56 @@
+module ToolBelt
+  module Command
+    class ProcedureCommand < Clamp::Command
+      subcommand "foreman", "Foreman procedures" do
+        subcommand "branch", "Start branching Foreman" do
+          parameter "release", "Version that the branch will have - like 1.20"
+
+          def execute
+            context = {
+              release: release,
+              develop: bump_last(release),
+            }
+
+            render('branch', context)
+          end
+        end
+
+        subcommand "release", "Start releasing Foreman" do
```

Comment:
Having a subcommand that has the same parameter name and is the same parameter name as the previous subcommand is confusing.  Maybe new-release? Or name the parameter version?

Reply:
You're right so I'm changing it to:
```
full_version: 1.20.0-RC1
debian_full_version: 1.20.0~rc1-1
version: 1.20.0
branch: 1.20
extra: RC1
```

It's still the release procedure but there's no more overlap.